### PR TITLE
(Feature) #1997 Dashboard should show world map with claim bubbles

### PR DESCRIPTION
**Pull request by yaroslav-fedyshyn-nordwhale**
# Description

Added new env variable to provide google data studio URL. Added iframe with google data studio URL to the dashboard.

About https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/1997

The link to the GoodDapp PR - https://github.com/GoodDollar/GoodDAPP/pull/2056

# How Has This Been Tested?

Open Dashboard and see the new Google data studio box.

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [x] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

## Screenshots:

| Browser | Simulators |
| --- | --- |
|  |  |

### CODE REVIEW

**Comment by LiavGut** [comment]
@sirpy @yaroslav-fedyshyn-nordwhale 
1. I've updated the design (Changed colors)
2. the label (on hover state) should be "country name" and the amount text should be "Claims"

Screen Shot 2020-06-22 at 13 12 08

https://datastudio.google.com/u/1/reporting/1BI7io5MQdlY-pgRtIwahM35XCEvTTWAu/page/mlwTB/edit

**Comment by serdiukov-o-nordwhale** [comment]
please add REACT_APP_EMBED_DATA_STUDIO_URL to the .env.development

File: client/src/config.js [hunk 1]
```diff
@@ -5,4 +5,5 @@ export default {
   walletUrl: env.REACT_APP_WALLET_URL,
   gunUrl: env.REACT_APP_GUN_URL,
   apiUrl: env.REACT_APP_API_URL,
+  embedDataStudioUrl: env.REACT_APP_EMBED_DATA_STUDIO_URL,
```
Comment (serdiukov-o-nordwhale) [inline client/src/config.js:8]: should be added to the env example (e.g. env.development in this package)
or better create separate .env.example containing all env vars available

**Review by serdiukov-o-nordwhale — CHANGES_REQUESTED** [review]

**Comment by yaroslav-fedyshyn-nordwhale** [comment]
> @sirpy @yaroslav-fedyshyn-nordwhale
> 
> 1. I've updated the design (Changed colors)
> 2. the label (on hover state) should be "country name" and the amount text should be "Claims"
> 
> Screen Shot 2020-06-22 at 13 12 08
> 
> https://datastudio.google.com/u/1/reporting/1BI7io5MQdlY-pgRtIwahM35XCEvTTWAu/page/mlwTB/edit

I don't know where to edit this view. It is cannot be configured with the Dashboard code. We just display the content of the specified URL.

@sirpy the Dashboard and GoodDapp part are done. What about new design?

**Review by serdiukov-o-nordwhale — APPROVED** [review]

**Pull request merged**